app/build.gradle

```diff
@@ -26,20 +26,55 @@ android {
     lintOptions {
         abortOnError false
     }
+    testOptions {
+        unitTests.returnDefaultValues = true
+    }
 }
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
-        exclude group: 'com.android.support', module: 'support-annotations'
-    })
+
     compile 'com.android.support:appcompat-v7:25.3.1'
     compile 'com.android.support:support-v4:25.3.1'
     compile 'com.android.support:design:25.3.1'
     compile 'com.android.support.constraint:constraint-layout:1.0.0'
+
     testCompile 'junit:junit:4.12'
-    // Optional -- Mockito framework
-    testCompile 'org.mockito:mockito-core:1.10.19'
+
+    //Android test compiles
+    androidTestCompile "com.google.dexmaker:dexmaker:1.2"
+    androidTestCompile "com.google.dexmaker:dexmaker-mockito:1.2"
+    androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
+    androidTestCompile "org.mockito:mockito-core:1.10.19"
```

Comment:
I had a look at the docs, it does seem we don't have this library. Created an issue here #123 

As far as mockito is concerned, you should add these test to the `test` folder and not `androidTest`.
Also make sure you have mentioned `@RunWith(MockitoJUnitRunner.class)` annotation for the test class